spec/component/models/activity_insight_oa_file_spec.rb

```diff
@@ -66,19 +66,27 @@
                          title: 'pub3',
                          open_access_locations: [
                            build(:open_access_location, source: Source::OPEN_ACCESS_BUTTON, url: 'url', publication: nil)
-                         ])
+                         ],
+                         open_access_status: 'green')
     }
     let!(:pub4) { create(:publication,
-                         title: 'pub4',
-                         open_access_status: nil)
+                         title: 'pub1',
```

Comment:
I'm guessing that you didn't mean to change these titles?

Reply:
Looks like pub5 and pub6 are like this too.